incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -312,3 +332,18 @@ func removeHNCConfigType(ctx context.Context, apiVersion, kind string) {
 		return updateHNCConfig(ctx, c)
 	}).Should(Succeed())
 }
+
+func createCronTabCRD(ctx context.Context) {
+	crontab := v1beta1.CustomResourceDefinition{}
+	crontab.Kind = "CronTab"
+	crontab.APIVersion = "stable.example.com/v1"
```

Comment:
Really? Are you sure this isn't `CustomResourceDefinition` and `apiextensions.k8s.io/v1beta` (or whatever) respectively? Eg see line 262 on this file.

Reply:
Interesting. It should be `CustomResourceDefinition` and `apiextensions.k8s.io/v1beta1` (I used the definition from [here](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)). When I tried to install the CRD on the apiserver, I got some errors complaining about invalid definition, but I never got an error for that.

Reply:
Could be that the Go client just overwrote the (obviously bad) fields for you.